testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -156,8 +176,17 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
     cluster_topology = params_from_base_suite_setup["cluster_topology"]
     mode = params_from_base_suite_setup["mode"]
     xattrs_enabled = params_from_base_suite_setup["xattrs_enabled"]
+    sg_lb = params_from_base_suite_setup["sg_lb"]
 
     test_name = request.node.name
+
+    if sg_lb:
+        # These tests target one SG node
+        skip_tests = ['online', 'offline', 'log_rotation', 'openidconnect']
+        for test in skip_tests:
+            if test in test_name:
+                pytest.skip("Skipping online/offline tests with load balancer")
```

Comment:
If we are going to skip these for now, we need at least a tracking ticket to bring the tests to feature parity.